CI/travis/package_darwin.sh

```
@@ -44,7 +44,7 @@ sudo install_name_tool -change ${libusbpath} @executable_path/../Frameworks/${li
 
 sudo /usr/local/opt/qt/bin/macdeployqt Scopy.app
 
-brew install python@2
+brew install python@2 || 1
```

this could hide an error;